x/docs/md/intro/getting-started.md

```diff
@@ -3,7 +3,7 @@
 1. Download and configure the [command-line client](/cli).
 1. Fetch the first exercise (`exercism fetch TRACK_ID`).
 1. Write a solution to make the test suite pass.
-1. Submit your solution (`exercism submit path/to/file`).
+1. Submit your solution (`exercism submit path/to/file` or `exercism s path/to/file{.h,.cpp}`). 
```

Comment:
Good question. I don't know if that works on the Mac. 

The only tracks that would need this in for is the ones that use headers c++, objectiveC , plain others? [1] Maybe this can go in those tracks doc? But where ? 

[1]
https://www.quora.com/Which-programming-languages-like-C-C++-separate-function-class-declarations-in-the-header-file-h-and-the-implementation-file-c-cpp